src/uncontrast/pages/UploadNewPaper/ExperimentsSection/ProcessingDomain.jsx

```diff
@@ -0,0 +1,120 @@
+import {
+  AddFieldButton,
+  ExpandingBox,
+  TooltipExplanation,
+  TrashButton,
+  CustomSelect,
+  CircledIndex,
+} from "../../../../sharedComponents/Reusble";
+import { useEffect, useState } from "react";
+import {
+  DeleteClassificationField,
+  SubmitClassificationField,
+  alphabetizeByLabels,
+  rawTextToShow,
+} from "../../../../Utils/functions";
+
+export default function ProcessingDomain({
+  fieldOptions,
+  disabled,
+  experiment_pk,
+  study_pk,
+  values,
+  setMinimumClassifications,
+  minimumClassifications,
+}) {
+  const initialValues = { main: "" };
+  const [fieldValues, setFieldValues] = useState([initialValues]);
+  const classificationName = "processing_domain";
+
+  const handleSubmit = SubmitClassificationField(
+    study_pk,
+    experiment_pk,
+    classificationName,
+    fieldValues,
+    setFieldValues
+  );
+
+  const handleDelete = DeleteClassificationField(
+    study_pk,
+    experiment_pk,
+    classificationName,
+    fieldValues,
+    setFieldValues
+  );
+  useEffect(() => {
+    if (values?.tasks && values.tasks.length > 0) {
+      setFieldValues(
+        values.tasks.map((row) => {
+          return {
+            main: row.main,
+            id: row.id,
+          };
+        })
+      );
+    }
+  }, []);
+  const fieldsNum = fieldValues.filter((field) => field.id)?.length;
+  useEffect(() => {
+    setMinimumClassifications({
+      ...minimumClassifications,
+      tasks: fieldsNum,
+    });
+  }, [fieldsNum]);
+
+  return (
+    <ExpandingBox
+      number={fieldsNum}
+      disabled={false} //TODO
+      headline={rawTextToShow(classificationName)}>
+      {fieldValues.map((fieldValue, index) => {
+        return (
+          <div key={`${classificationName}-${index}`}>
```

Comment:
Its better not to use index as key or part of key in React because if the order changes than the diffing mechanism can fail